config/dev/onebox.yml

```diff
@@ -63,6 +63,7 @@ EDXAPP_CMS_SSL_NGINX_PORT: 48010
 EDXAPP_CMS_SSL_NGINX_FRONTEND_PORT: 443
 
 NGINX_ENABLE_SSL: %%NGINX_ENABLE_SSL%%
+NGINX_REDIRECT_TO_HTTPS: %%NGINX_ENABLE_SSL%%
```

Comment:
I actually make a similar fix in #284 by moving NGINX_REDIRECT_TO_HTTPS into the shared server-vars.yml file. I prefer my solution since it reduces duplication. Do you mind reverting onebox.yml in favor of #284 ?

Reply:
Will do.